### ISSUE TITLE
Provide list of hardware issues the 58 FC430s from R1-PC-C20 and repair any found issues

### ISSUE THREAD

**Issue by joachimweyl** [nerc-project/operations#218]
Review the remaining 58 FC430 nodes in R1-PA-C08 (88 - 20 already in OpenShift, 10 for Observability).
create a list of any hardware issues so that we can replace them with other FC430s we have from the Wayfair donation.

**Comment by hakasapl** [comment]
@joachimweyl I will see if I can do this tomorrow if I have time, but there are other priorities at MGHPCC tomorrow. This will most likely land next sprint.

**Comment by aabaris** [comment]
@hakasapl and @joachimweyl please be aware that:
```
cmc-9-obm.nerc-ocp-prod.rc.fas.harvard.edu
Module ID     = server-1d  memory error rate exceeded for DIMM_A3
```
is tied to:  wrk-75 which is allocated to for hypershift testing.

Please co-ordinate poweroff and repair of this specific module with @larsks

**Comment by msdisme** [comment]
@hakasapl seems like a good use of techsquare

**Comment by joachimweyl** [comment]
@hakasapl is this on techsquared list?

**Comment by joachimweyl** [comment]
@hakasapl do we have an expected completion date from Techsquared?

**Comment by hakasapl** [comment]
I have not sent this over yet as techsquare are working on a few other things for us right now, I will send it over once their plate clears up

**Comment by hakasapl** [comment]
@aabaris I'll engage with Techsquare and link them this issue. I'll turn this over to techsquare after [this issue](https://github.com/nerc-project/operations/issues/313)

**Comment by hakasapl** [comment]
@aabaris could you update this issue with the physical node locations for these machines, as well as what their status is? ie. can techsquare take them offline to do maintenance?

**Comment by aabaris** [comment]
> @aabaris could you update this issue with the physical node locations for these machines, as well as what their status is? ie. can techsquare take them offline to do maintenance?

Yes @hakasapl My ETA for providing this info is sometime early tomorrow 2023-11-30 (I'm multitasking through a day of almost non stop meetings)  Thank you for your help with this!

**Comment by aabaris** [comment]
> > @aabaris could you update this issue with the physical node locations for these machines, as well as what their status is? ie. can techsquare take them offline to do maintenance?
> 
> Yes @hakasapl My ETA for providing this info is sometime early tomorrow 2023-11-30 (I'm multitasking through a day of almost non stop meetings) Thank you for your help with this!

My ETA is not working out :(  but I'm actively working on this.  If there are timeline  priorities from your end, let me know.  My plan is to provide a map of HW issues, combined with locations and indication of which hardware has been allocated to what clusters/projects.

**Comment by aabaris** [comment]
So far I have identified the following hardware issues:
--------------------------
location: r1-pc-20 u11
device: FX chassis(named cmc-2)
fault: fan-1 is critical
status: nodes in the chassis are actively in production, but fan might be hot swappable
---------------------------
location: r1-pc-20 u13
device: Server in slot-3a (in FX chassis named cmc-3)
fault: cmos battery fail
status: node is in production, offlining needs to be scheduled to repair
---------------------------
location: r1-pc-20 u13
device: Server in slot-3c (in FX chassis named cmc-3)
fault: cmos battery fail
status: node is in production, offlining needs to be scheduled to repair
---------------------------
location: r1-pc-20 u19
device: FX chassis(named cmc-6)
fault: fan-5 is critical
status: nodes currently not in production
---------------------------
location: r1-pc-20 u19
device: Server in slot-3a (in FX chassis named cmc-6)
fault: cmos battery fail
status: nodes currently not in porduction
---------------------------
location: r1-pc-20 u21
device: FX chassis(named cmc-7)
fault: fan-1 and fan-5 are critical
status: nodes currently not in production
---------------------------

**Comment by yani4** [comment]
@aabaris Do you have replacement fans for the machines that need them or the cmos batteries?

I was taking a look at these chassis trying to figure out how removing the fans while in production and unfortunately the power cords look too short to hot swap them.

**Comment by joachimweyl** [comment]
@hakasapl correct me if I am wrong but I don't believe the nodes with issues are currently in use. Also, do we have the fans/CMOS for these machines in the cage?

**Comment by hakasapl** [comment]
Yes, we have some spare fans in the fx2 chassis sitting in the spare parts rack in the back. Flax is doing some work on consolidation back there today through Tuesday so I will get you these parts next week.

**Comment by hakasapl** [comment]
@aabaris or @jtriley do you know if the whole chassis on these nodes can be pulled?

**Comment by aabaris** [comment]
> @aabaris Do you have replacement fans for the machines that need them or the cmos batteries?
> 
> I was taking a look at these chassis trying to figure out how removing the fans while in production and unfortunately the power cords look too short to hot swap them.

I believe @hakasapl  would have access to spare parts if any are available.

**Comment by aabaris** [comment]
> @aabaris or @jtriley do you know if the whole chassis on these nodes can be pulled?

I the list I provided "status" line indicates if devices are in production or not, if they are in production and can not be fixed with just hot swapping or working around active components, then we'll need to schedule some form downtime in order to do reparis.

**Comment by yani4** [comment]
Ok good to know on the topic of parts.

Having looked at them the cabling is too short to attempt a hotswap in the case of the fans due to their location on the server. Let me know when downtime is scheduled and we can work with the machines.

**Comment by hakasapl** [comment]
@aabaris I am going to close this since it's very old, if we still have issues to resolve regarding hardware in NERC please open issues by-node using the new template
